harness/determined/cli/rbac.py

```diff
@@ -150,9 +207,24 @@ def list_groups_roles(args: Namespace) -> None:
     if resp.roles is None or len(resp.roles) == 0:
         print("group has no role assignments")
         return
-    render.render_objects(
-        v1RoleHeaders, [render.unmarshal(v1RoleHeaders, r.to_json()) for r in resp.roles]
-    )
+
+    for i, r in enumerate(resp.roles):
+        assigned_globally = ""
+        if resp.assignments[i].isGlobal:
+            assigned_globally = " globally and"
+        print(f"role '{r.name}' with ID {r.roleId} assigned{assigned_globally} to workspaces")
```

Comment:
non blocking: I wondering if it's better to have the global scope be another object in the workspaces list vs as a sentence saying "assigned globally and to workspaces"